### PR TITLE
fix: Remove proposal pop-up and show options below cfs

### LINKED ISSUE #4587: Call for Speakers: Show option below CfS instead of pop-up
Currently the CfS provides a pop-up. Instead of a pop-up show the options below the CfS:
1. Show info "No session have been submitted." (as long as there are no existing sessions)
2. Show the button "Add Speaker Details" (in the same way as of now, but below the CfS)
3. Show the option "Submit Session" also below the CfS, but if the user has not added speaker details yet, show an error message "You need to add your speaker details first before submitting a session".

Screenshot from 2020-07-20 20-39-22

### PR DESCRIPTION
**Pull request by Haider8**
Fixes #4587 

#### Short description of what this resolves:
There's a proposal pop-up in cfs which we don't want anymore

#### Changes proposed in this pull request:
Show options below cfs

When cfs is closed:

when-cfs-closed

When cfs is open:

when-cfs-open

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Haider8** [comment]
@iamareebjamal How to show error message when the user tries to add new session before adding his/her speaker details?

**Comment by iamareebjamal** [comment]
You shouldn't show the button to add session details if speaker details haven't been added

**Comment by Haider8** [comment]
Yes but Mario was saying in the issue that
> Show the option "Submit Session" also below the CfS, but if the user has not added speaker details yet, show an error message "You need to add your speaker details first before submitting a session".

**Comment by iamareebjamal** [comment]
Then use `this.notify`

File: app/components/public/call-for-speakers.js [hunk 1]
```diff
@@ -13,6 +13,18 @@ export default class CallForSpeakers extends Component {
     }
   }
 
+  @action
+  addSession() {
+    if (this.data.userSpeaker && this.data.userSpeaker.toArray().length) {
+      // speaker detail exists
```
Comment (Haider8) [inline app/components/public/call-for-speakers.js:19]: @iamareebjamal Should I add `this.transitionToRoute` here for directing towards `public.cfs.new-session` route?

Comment (iamareebjamal) [reply app/components/public/call-for-speakers.js:19]: Yes

**Comment by Haider8** [comment]
@iamareebjamal Is there a need to change message text of LoginSignupModal? Currently it is "Please Login or Signup before you can submit a proposal.". I was thinking that it can now be "Please Login or Signup before adding your speaker details." because this modal is now opening when unauthenticated user clicks on "Add Speaker Details" button.

**Comment by iamareebjamal** [comment]
@mariobehling Please check what should be done

**Comment by mariobehling** [comment]
> @iamareebjamal Is there a need to change message text of LoginSignupModal? Currently it is "Please Login or Signup before you can submit a proposal.". I was thinking that it can now be "Please Login or Signup before adding your speaker details." because this modal is now opening when unauthenticated user clicks on "Add Speaker Details" button.

No need to change the message as proposal could refer to the broader submission action.

**Comment by Haider8** [comment]
@iamareebjamal The PR is now final. Is there anything else?

**Pull request merged**